internal/pkg/service/appsproxy/appconfig/appconfig.go

```diff
@@ -37,6 +41,11 @@ type cacheItem struct {
 	updateLock *sync.Mutex
 }
 
+type notificationItem struct {
+	lastNotification *time.Time
```

Comment:
If you use `nextNotificationAfter time` instead of `lastNotification *time.Time`, the code will be simple:
- No `nil` value, `if now.After(nextNotificationAfter)` will work also with the empty value.